flow-typed/custom/jsdom.js

```diff
@@ -0,0 +1,320 @@
+// flow-typed signature: 6393943152251b303dd7a2ad2a2fff94
```

Comment:
This file is based on the typescript def with some changes.

Reply:
Very cool!